csp/gitprojectsettings.csp

```diff
@@ -263,6 +275,26 @@ body {
             </div>
         </div>
         
+        <div class="form-group row mb-3">
+            <label for="systemBasicMode" class="offset-sm-1 col-sm-3 col-form-label" data-toggle="tooltip" data-placement="top" title="Whether basic mode should be applied as the system-wide default"> Basic Mode as System Default</label>
```

Comment:
Reword:
Enable basic mode for all users, which reduces the number of menu options and requires less knowledge of git. This is suitable for a single-developer (non-shared) namespace.